test/broadcast.jl

```diff
@@ -45,32 +45,32 @@ end
     @testset "2x2 StaticMatrix with 1x2 StaticMatrix" begin
         m1 = @SMatrix [1 2; 3 4]
         m2 = @SMatrix [1 4]
-        @test_broken @inferred(broadcast(+, m1, m2)) === @SMatrix [2 6; 4 8] #197
-        @test_broken @inferred(m1 .+ m2) === @SMatrix [2 6; 4 8] #197
+        @test @inferred(broadcast(+, m1, m2)) === @SMatrix [2 6; 4 8] #197
+        @test @inferred(m1 .+ m2) === @SMatrix [2 6; 4 8] #197
```

Comment:
Perhaps it makes sense to remove the issue reference from these individually, and instead describe the behaviour which is being tested in a short comment.  (That comment could include the issue crossref instead.)

In my opinion the source code (including comments) should be the source of truth about design decisions and the reasons for testing given behaviour: the source is a living document, but issues become abandoned as soon as they're closed.